src/app/components/OrderBook.tsx

```diff
@@ -130,16 +130,31 @@ export function OrderBook() {
   );
   const sells = useAppSelector((state) => state.orderBook.sells);
   const buys = useAppSelector((state) => state.orderBook.buys);
-  //const grouping = useAppSelector((state) => state.orderBook.grouping);
+  const [showOrderBookLabel, setShowOrderBookLabel] = useState(true);
+
+  useEffect(() => {
+    const handleResize = () => {
+      setShowOrderBookLabel(window.innerWidth > 1400);
```

Comment:
This isn't ideal as all screensizes <1400 will not show the title, even if there is enough space (e.g. mobile):

![image](https://github.com/DeXter-on-Radix/website/assets/44790691/4836f76a-45df-44ab-9035-db2b61e4bdbf)